docker-compose/monitor/docker-compose.yml

```diff
@@ -22,8 +22,6 @@ services:
     volumes:
       - ${OTEL_CONFIG_SRC:-./otel-collector-config-connector.yml}:/etc/otelcol/otel-collector-config.yml
     command: --config /etc/otelcol/otel-collector-config.yml
-    ports:
-      - "4317:4317"
```

Comment:
Note that I've also removed this as we're executing tracegen in the same `monitor_backend` network. Also tested this locally to make sure:

```shell
# Fail to access the port from host machine.
$ telnet localhost 4317
Trying 127.0.0.1...
telnet: Unable to connect to remote host: Connection refused

# Successfully write to port from docker command.
$ docker run --env OTEL_EXPORTER_OTLP_TRACES_ENDPOINT="http://otel_collector:4317" \
  --network monitor_backend \
  --rm \
  jaegertracing/jaeger-tracegen:1.49 \
    -trace-exporter otlp-grpc \
    -traces 1
...
2023-09-12T20:40:01.903Z	LEVEL(-8)	global/internal_logging.go:62	exporting spans	{"count": 2, "total_dropped": 0}
```